hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/sigs/HollowScreening.java

```diff
@@ -170,11 +174,19 @@ private static List<PendingCompletion> maybeAddToCompletions(
             final byte[] evmAddress,
             final JECDSASecp256k1Key correspondingKey,
             List<PendingCompletion> pendingCompletions,
-            final AliasManager aliasManager) {
-        final var accountNum = aliasManager.lookupIdBy(ByteStringUtils.wrapUnsafely(evmAddress));
+            final AliasManager aliasManager,
+            @Nullable final LinkedRefs linkedRefs) {
+        final var alias = ByteStringUtils.wrapUnsafely(evmAddress);
+        if (linkedRefs != null) {
+            linkedRefs.link(alias);
+        }
+        final var accountNum = aliasManager.lookupIdBy(alias);
         // a hollow account cannot be CryptoDelete-d, but it may have expired since the latest
         // immutable state
         if (accountNum != EntityNum.MISSING_NUM) {
+            if (linkedRefs != null) {
+                linkedRefs.link(accountNum.longValue());
+            }
```

Comment:
Do we need to link the num here? The `StateChildrenSigMetadataLookup` calls during account key collection in `Expansion`/`Rationalization` , which happens before the hollow screening, [will have already linked the account num, correct? ](https://github.com/hashgraph/hedera-services/blob/c96ec62c77d2764fd4a499983639846593a2edae/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/sigs/metadata/StateChildrenSigMetadataLookup.java#L187-L191)  Maybe we want to be explicit about it and not take any chances, I don't think there will be any problems with having duplicates in the linked refs?

Reply:
Good point, it's not really necessary here afaict; but doesn't hurt to be explicit I guess. 🤔